### PR TITLE
medaka: allow gz input

### PR DESCRIPTION
**Pull request by bernt-matthias**
I tried to bump to 1.2.3 .. but this gives errors .. just in case someone wants to jump in

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/medaka/macros.xml [hunk 1]
```diff
@@ -29,8 +29,13 @@
...
     <token name="@REF_FASTA@"><![CDATA[
         #if $reference_source.reference_source_selector == 'history':
-            ln -f -s '$reference_source.ref_file' reference.fa &&
+            #if $reference_source.ref_file.ext.endswith(".gz")
+                gunzip -c '$reference_source.ref_file' > reference.fa &&
+            #else
+                ln -f -s '$reference_source.ref_file' reference.fa &&
+            #end if
         #else:
+            #set reference="reference.fa"
```
Comment (bgruening) [inline tools/medaka/macros.xml:38]: why is this needed?

**Review by bgruening — COMMENTED** [review]
Do you want to add to all those PRs the suffix variable?

**Comment by bernt-matthias** [comment]
> Do you want to add to all those PRs the suffix variable?

Thought about this. But I'm wondering if we want to do this for the suite.

File: tools/medaka/macros.xml (hunk 1 above)
Comment (bgruening) [reply tools/medaka/macros.xml:38]: ping ...

**Review by bgruening — APPROVED** [review]
